README.md

```diff
@@ -1,70 +1,86 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Book Store
 
-## Available Scripts
+## About
 
```

Comment:
Please add the `npm start` command to README so the user is able to start

Reply:
Thank you. I add it.